scenario/GovernanceScenario.ts

```diff
@@ -147,3 +151,53 @@ scenario('add new asset',
     expect(await albert.getCometCollateralBalance(dogecoin.address)).to.be.equal(exp(100, 8));
     expect(await albert.getCometBaseBalance()).to.be.equal(-borrowAmount);
   });
+
+scenario(
+  'execute Mumbai governance proposal',
+  {
+    filter: async (ctx) => ctx.world.base.network === 'mumbai'
+  },
+  async ({ timelock, bridgeReceiver }, _context, world) => {
+    const governanceDeploymentManager = world.auxiliaryDeploymentManager;
+    if (!governanceDeploymentManager) {
+      throw new Error("cannot execute governance without governance deployment manager");
+    }
+
+    const proposer = await impersonateAddress(governanceDeploymentManager, COMP_WHALES.testnet[0]);
```

Comment:
You can just call `await context.getProposer()`

Reply:
if i'm not mistaken, `context.getProposer` would return you the signer for Mumbai, but here we need the signer for the governance chain, Goerli

Reply:
Ah I see, that's right